Makefile

```diff
@@ -633,6 +633,19 @@ generate-nomad-rust-client: ## Generate the Nomad rust client from OpenAPI
 # should use that here.
 	$(MAKE) format-prettier
 
+.PHONY: buildx-tracing
+buildx-tracing: ## Set tracing up for docker buildx, this assumes that Jaeger is already up
+	docker buildx create \
+      --name builder \
```

Comment:
It'd be good to name this something more descriptive, like "grapl-tracing", or "grapl-jaeger", etc.

Reply:
So since this only sets tracing for docker (we set up tracing for the grapl services without this), I think it makes more sense to rename this docker-tracing than grapl-tracing.

Reply:
I meant that, among the _n_ potential builders you might one day have on your system, this would be the one that Grapl has set up for tracing builds. It's inherently for Docker since it's a Docker builder 🙃

Reply:
Ah, you meant the actual builder name, not the make command 🤦 . Fixing now